UNIFY-328 Update all references to config file (#4093)
cypress-io/cypress-documentation: content/guides/guides/command-line.md

</Alert>

#### `cypress run --config-file <configuration-file>`

You can specify a path to a file where

> [configuration](/guides/references/configuration) values are set.

Reviewer: ```suggestion
[Cypress configuration](/guides/references/configuration) values are set.
```

Options passed to `cypress open` will automatically be applied to the project
you open. These persist on all projects until you quit the Cypress Test Runner.

> These options will also override values in your configuration file.

Reviewer: ```suggestion
These options will also override values in the Cypress configuration file.
```

cypress run --config pageLoadTimeout=100000,watchForFileChanges=false
```

#### `cypress open --config-file <configuration-file>`

You can specify a path to a file where

> [configuration](/guides/references/configuration) values are set.

Reviewer: ```suggestion
[Cypress configuration](/guides/references/configuration) values are set.
```
